src/views/projects/Issue/New.svelte

```diff
@@ -164,32 +85,21 @@
 <main>
   <div class="form">
     <div class="editor">
-      {#if preview}
-        <div class="summary txt-medium" class:txt-missing={preview}>
-          {#if issueTitle.trim() === ""}
-            No title
-          {:else}
-            {issueTitle}
-          {/if}
-        </div>
-        <div class="comments">
-          <Comment
-            rawPath={project.getRawPath()}
-            comment={{
-              author: { id: session.publicKey },
-              body: issueText,
-              reactions: {},
-              replyTo: null,
-              timestamp: Date.now(),
-            }} />
-        </div>
-      {:else}
-        <Textarea bind:value={issueTitle} placeholder="Title" />
-        <Textarea
-          bind:value={issueText}
-          resizable
-          placeholder="Leave a comment" />
-      {/if}
+      <IssueHeader
+        id=""
```

Comment:
Could we change the `id` type to `string | undefined` instead of passing and empty string here?